pystac/extensions/projection.py

```diff
@@ -480,6 +480,10 @@ def migrate(
         if epsg := obj["properties"].pop("proj:epsg", None):
             obj["properties"]["proj:code"] = f"EPSG:{epsg}"
 
+        for asset in obj.get("assets", obj.get("item_assets", {})).values():
+            if epsg := asset.pop("proj:epsg", None):
+                asset["proj:code"] = f"EPSG:{epsg}"
+
```

Comment:
I think this will skip `item_assets` if `assets` is present ... presumably we want to migrate both?

Reply:
Good catch! I'll fix this.